README.md

```diff
@@ -45,21 +45,30 @@ $ waybar
 gtkmm3
 jsoncpp
 libinput
-
 libsigc++
 fmt
 wayland
-wlroots
+chrono-date
+spdlog
 libgtk-3-dev [gtk-layer-shell]
 gobject-introspection [gtk-layer-shell]
 libgirepository1.0-dev [gtk-layer-shell]
 libpulse [Pulseaudio module]
 libnl [Network module]
-sway [Sway modules]
```

Comment:
Why removing sway?

Reply:
I see this list being used by people who want to package waybar for different operating system, and having sway as a dependency says as if waybar cannot physically run without sway, as if it depends on some shared libraries from sway - which it doesn't. Same for wlroots dependency, as far as I could see waybar doesnt actually depend on anything from wlroots. More details in the description of the PR :)